Formula/nethack.rb

```diff
@@ -53,4 +62,18 @@ def install
     chmod "g+w", libexec
     chmod "g+w", libexec+"save"
   end
+
+  if MacOS.version >= :mojave
+    def caveats
+      <<~EOS
+        You can activate the new curses interface using
+        NETHACKOPTIONS=windowtype:curses on the CLI or by editing your
+        options file at `~/Library/Preferences/NetHack Defaults.txt`
```

Comment:
Is this file read on the Terminal version? This should just be simplified to mention `.nethackrc` and ideally be copy and pasteable. Something like `To enable the curses interface, run: ...`

Reply:
IIUC, `.nethackrc` is not read at all on macOS and only the `~/Library/…` file is. It may be possible that both are read but the `Library` one is preferred?

I'm not sure what to compress here. I would like to mention both the 'runtime' option of `NETHACKOPTIONS` and the location of the NetHack options file (a location that's been somewhat difficult to look up in the past). I suppose I could mention the specific edit that would be needed in the options file (`OPTIONS=windowtype:curses`) but that would only serve to make the caveat longer?

Reply:
> IIUC, `.nethackrc` is not read at all on macOS and only the `~/Library/…` file is. It may be possible that both are read but the `Library` one is preferred?

`.nethackrc`, if it exists, is read prior to the library preferences file.

> I'm not sure what to compress here. I would like to mention both the 'runtime' option of `NETHACKOPTIONS` and the location of the NetHack options file (a location that's been somewhat difficult to look up in the past). I suppose I could mention the specific edit that would be needed in the options file (`OPTIONS=windowtype:curses`) but that would only serve to make the caveat longer?
> 
> > You can activate the new curses interface using `NETHACKOPTIONS=windowtype:curses` on the CLI or by adding the following line to your options file at `…`: `OPTIONS=windowtype:curses`.
> 
> Maybe?

See `brew info gdb` for what I mean.